### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by guillermoap**
### Summary
Created github actions workflow and build badge

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        gem install bundler:1.17.3
+        bundle _1.17.3_ update
+        bundle _1.17.3_ install --gemfile gemfiles/${{ matrix.gemfile }} --jobs 4 --retry 3
+        bundle _1.17.3_ exec rake code_analysis
+        bundle _1.17.3_ exec rspec
+    - name: Report to CodeClimate
+      run: |
+        ./cc-test-reporter after-build --exit-code 0
+      env:
+        CC_TEST_REPORTER_ID: 7196b4aa257fde33f24463218af32db6a6efd23d9148204822f757fa614a093e
```
Comment (martinjaimem) [inline .github/workflows/ci.yml:45]: wandering if we could move this up as we reference this twice 🤔

File: gemfiles/rails_head.gemfile [hunk 2]
```diff
@@ -2,6 +2,6 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails'
```
Comment (santib) [inline gemfiles/rails_head.gemfile:5]: I think this was fine as it was, it's the gemfile for the unreleased version of rails

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/rootstrap/active-storage-base64.svg?branch=master)](https://travis-ci.org/rootstrap/active-storage-base64)
```
Comment (santib) [inline README.md:1]: I think we should also delete `.travis.yml` and any other references to Travis (if any)

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        chmod +x ./cc-test-reporter
+        ./cc-test-reporter before-build
+    - name: Build and test with Rake
+      run: |
+        gem install bundler:1.17.3
+        bundle _1.17.3_ update
+        bundle _1.17.3_ install --gemfile gemfiles/${{ matrix.gemfile }} --jobs 4 --retry 3
+        bundle _1.17.3_ exec rake code_analysis
+        bundle _1.17.3_ exec rspec
```
Comment (santib) [inline .github/workflows/ci.yml:39]: not sure why is this needed but 👌

File: gemfiles/rails_head.gemfile (hunk 2 above)
Comment (guillermoap) [reply gemfiles/rails_head.gemfile:5]: Was getting this error both in CI and locally
image

Comment (santib) [reply gemfiles/rails_head.gemfile:5]: yes that's because we'd need to make the dependency on rails be more flexible (maybe just > x and not set a max version?). Otherwise you can just skip that test for now, but the gemfile is fine as it is

File: .github/workflows/ci.yml (hunk 4 above)
Comment (guillermoap) [reply .github/workflows/ci.yml:39]: will try it out without it

**Review by santib — APPROVED** [review]

**Review by martinjaimem — APPROVED** [review]
We should double-check the new behavior regarding the rails version used.

LGTM

**Review by santib — APPROVED** [review]
